lib/linux_admin/volume.rb

```diff
@@ -1,6 +1,5 @@
 module LinuxAdmin
   class Volume
-    private
 
     def self.process_volume_display_line(line)
```

Comment:
Same...Can you make this private_class_method (as well as elsewhere this type of change was made?)

Reply:
I did as many as I could without breaking the specs.